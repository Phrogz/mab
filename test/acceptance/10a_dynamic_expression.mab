p( "Test#10a" )
setSlot( "myExpr", Expression new( p( "Hello, world!" ) ) )
myExpr eval