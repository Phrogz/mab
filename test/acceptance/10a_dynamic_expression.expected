Test#10a
Hello, world!